### ISSUE TITLE
VICE test drive/defaults.prg

### ISSUE THREAD

**Issue by dirkwhoffmann** [davidhorrocks/hoxs64#1]
I was investigating why VIA1::T1L-L is marked in red when running test case drive/defaults.prg

![defaults](https://user-images.githubusercontent.com/12561945/43990127-7347093a-9d57-11e8-850a-cc9123f74a7c.png)

The test case requires 0x05 to pass. I understand that the latches are not reset, so 0x05 might just be some arbitrary value on startup. Is there a reason why you've chosen 0xAA (pattern 10101010) as the default value for T1L-L?

**Comment by davidhorrocks** [comment]
Hi Dirk,
Sometime ago, I did read the value $AA from another drive with was either a 1541-II or a 1571 that I do not use now. My current drive which is an another 1541-II prefers the value $D1 (reversed circle) as per my picture but sometimes it is $D3 (reversed heart).

The orbservant disk mag demo here [orbservant 1.zip](https://github.com/davidhorrocks/hoxs64/files/2280872/orbservant.1.zip) appears sensitive to this value. This demo occasionally fails on real hardware but will most likely fail with some values for the timer latch low. If you can press space bar at the first title screen to load the second main Orbservant title screen then you have passed the test. The failure scenario is to crash after pressing space.

![defaults](https://user-images.githubusercontent.com/9464356/44000867-7b4ca91e-9e1f-11e8-8727-62ff6dcc876e.jpg)

**Comment by dirkwhoffmann** [comment]
Hi David,
thanks a lot! I expected that there is some reason for the default value you have chosen. 

BTW, "observant" seems to hang in the second screen when it shows the "Editorial" stuff. Is this a bug in the observant program?

**Comment by davidhorrocks** [comment]
Hi Dirk,
I know that this title sometimes fails on real hardware but it works for me 90% of the time. I believe it contains a software bug where it fails to handle interrupts after you press space where you see a cursor blink rapidly in the top left corner which may cause a crash or corruption. I just tested this exact D64 on a real 1541-II and real C64C. I was able to load and read all news titles. I used a 1541 Ultimate II+ (running Retro Replay) to copy the D64 file to a real 1541-II drive.
